reconcile/glitchtip/reconciler.py

```diff
@@ -54,6 +62,21 @@ def _reconcile_projects(
                 new_project = Project(name=project.name, platform=project.platform)
             organization_projects.append(new_project)
 
+        for project in [p.desired for p in project_diff.change.values()]:
```

Comment:
can use iterator to avoid creating a new list

```python
for project in (p.desired for p in project_diff.change.values()):
```

Reply:
Good spot! Applied in f7aeaa5